core/src/main/java/com/linecorp/armeria/common/util/EventLoopThreadFactory.java

```diff
@@ -82,47 +90,23 @@ public EventLoopThreadFactory(String threadNamePrefix, boolean daemon, int prior
      * @param daemon whether to create a daemon thread.
      * @param priority the priority of the threads created by this factory.
      * @param threadGroup the {@link ThreadGroup}.
+     *
      */
     public EventLoopThreadFactory(String threadNamePrefix, boolean daemon, int priority,
                                   @Nullable ThreadGroup threadGroup) {
-        this(new EventLoopThreadFactoryImpl(requireNonNull(threadNamePrefix, "threadNamePrefix"),
-                                            daemon, priority, threadGroup));
+        super(requireNonNull(threadNamePrefix, "threadNamePrefix"), daemon, priority, threadGroup,
+              Function.identity());
     }
 
-    private EventLoopThreadFactory(ThreadFactory delegate) {
-        this.delegate = delegate;
+    EventLoopThreadFactory(String threadNamePrefix, boolean daemon, int priority,
+                           @Nullable ThreadGroup threadGroup,
+                           Function<? super Runnable, ? extends Runnable> taskFunction) {
+        super(requireNonNull(threadNamePrefix, "threadNamePrefix"), daemon, priority, threadGroup,
+              requireNonNull(taskFunction, "taskFunction"));
```

Comment:
`requireNonNull(taskFunction, "taskFunction")` -> `taskFunction`?

Reply:
Fixed it. Thank you.